app/components/public/ticket-list.js

```diff
@@ -40,6 +40,25 @@ export default Component.extend(FormMixin, {
     ) > 0;
   }),
 
+  donationTickets: computed('tickets', function() {
+    return this.tickets.filterBy('type', 'donation');
+  }),
+
+  isDonationPriceValid: computed('donationTickets', 'donationTickets.@each.orderQuantity', 'donationTickets.@each.price', function() {
```

Comment:
This might not be the most elegant way but found it working through this, also please resolve the proxy objects so that we are not using `.get()` Check this https://github.com/fossasia/open-event-frontend/pull/3078/files#r294945298
```js
donationPriceValid: computed('tickets.@each.orderQuantity', 'tickets.@each.price', function() {
    let returnValue = false;
    this.donationTickets.forEach(donationTicket => {
      if (donationTicket.get('orderQuantity') >= 0) {
        if (donationTicket.get('price') > 0) {
          returnValue = true;
        } else {
          returnValue = false;
        }
      }
    });
    return returnValue;
  }),
```